### PR TITLE
added profile:

### PR DESCRIPTION
**Pull request by sidharthviz**
Profile addition

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,7 @@ git push -u origin <your_branch_name>
 | 24.     | [Tamanna Sharma](https://github.com/Tamanna-S) | [Twitter](https://twitter.com/__tanan__) | i am in love with javaScript!! | JavaScript! |
 
 <!-- Add your names here -->
+| 26.     | [Siddharth Verma](https://github.com/sidharthviz) | [LinkedIn](https://www.linkedin.com/in/siddharth-verma-2695ab192/) | I like coding, Books & Travel enthusiast |
```
Comment (Panquesito7) [inline README.md:185]: The number should be 25 and above the comment.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sidharthviz) [reply README.md]: Thank you.  Changes Done

File: README.md [hunk 2]
```diff
@@ -181,6 +181,9 @@ git push -u origin <your_branch_name>
 | 23.     | [Shubham Patrick](https://github.com/AHTHneeuhl/)           | [LinkedIn](https://www.linkedin.com/in/ahthneeuhl/)             | Books & Music                                        | JavaScript & Python       |
 | 24.     | [Tamanna Sharma](https://github.com/Tamanna-S) | [Twitter](https://twitter.com/__tanan__) | i am in love with javaScript!! | JavaScript! |
 
+| 25.     | [Siddharth Verma](https://github.com/sidharthviz) | [LinkedIn](https://www.linkedin.com/in/siddharth-verma-2695ab192/) | I like coding, Books & Travel enthusiast |
+
+
```
Comment (Panquesito7) [inline README.md:186]:
```suggestion
| 25.     | [Siddharth Verma](https://github.com/sidharthviz) | [LinkedIn](https://www.linkedin.com/in/siddharth-verma-2695ab192/) | I like coding, Books & Travel enthusiast |
```

File: README.md [hunk 3]
```diff
@@ -180,7 +180,7 @@ git push -u origin <your_branch_name>
 | 22.     | [Sai Ram](https://github.com/ps1854) | [LinkedIn](https://www.linkedin.com/in/sai-ram-pendyala/) | I Love Music 🎵 | Python 👨‍💻 |
 | 23.     | [Shubham Patrick](https://github.com/AHTHneeuhl/)           | [LinkedIn](https://www.linkedin.com/in/ahthneeuhl/)             | Books & Music                                        | JavaScript & Python       |
 | 24.     | [Tamanna Sharma](https://github.com/Tamanna-S) | [Twitter](https://twitter.com/__tanan__) | i am in love with javaScript!! | JavaScript! |
-
+| 25.     | [Siddharth Verma](https://github.com/sidharthviz) | [LinkedIn](https://www.linkedin.com/in/siddharth-verma-2695ab192/) | I like coding, Books & Travel enthusiast |
```
Comment (Panquesito7) [inline README.md:183]:
```suggestion
| 25.     | [Siddharth Verma](https://github.com/sidharthviz) | [LinkedIn](https://www.linkedin.com/in/siddharth-verma-2695ab192/) | I like coding, Books & Travel enthusiast |

```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**